src/Tizen.MachineLearning.Inference/Interop/Interop.Nnstreamer.cs

```diff
@@ -23,6 +23,8 @@ internal static partial class Interop
     internal static partial class Libraries
     {
         public const string Nnstreamer = "libcapi-nnstreamer.so.1";
+        public const string Ml_common = "libcapi-ml-common.so.1";
+        public const string Ml_single = "libcapi-ml-inference-single.so.1";
```

Comment:
사소한것이긴 하지만, TizenFX에서는 파스칼 표기법을 선호하고 있습니다.
```suggestion
        public const string MlCommon = "libcapi-ml-common.so.1";
        public const string MlSingle = "libcapi-ml-inference-single.so.1";
```

Reply:
@myroot 감사합니다. 말씀해주신 사항 반영하여 업데이트했습니다.